pkg/apis/ops/v1alpha1/types.go

```diff
@@ -189,3 +222,47 @@ type TraceflowList struct {
 
 	Items []Traceflow `json:"items"`
 }
+
+// GetDstType gets the name of destination for specific traceflow.
+func (tf *Traceflow) GetDstName() string {
```

Comment:
I think such method is helpful but not necessary and you may put them somewhere else. What do you think?

Reply:
As @gran-vmv proposed, GetTraceflowMessage can be remained in types.go. The other functions could be moved to somewhere else.

Reply:
+1

Reply:
+1

Reply:
Moved to somewhere else.